features/fixtures/business_readiness.json

```diff
@@ -33,6 +33,353 @@
         "web_url": "https://www.gov.uk/find-eu-exit-guidance-business/email-signup"
       }
     ],
+    "ordered_related_items": [
+      {
+        "api_path": "/api/content/guidance/the-food-and-drink-sector-and-preparing-for-eu-exit",
```

Comment:
Do we need all the details here for the ordered related items? Would it work with just the minimum? Only because it makes the json file even longer and a bit difficult to read.

Reply:
I'll shrink them down to just two.

Reply:
@vanitabarrett I've amended and fixed up.